code/game/objects/items/food/pastries.dm

```diff
@@ -0,0 +1,779 @@
+//Pastry is a food that is made from dough which is made from wheat or rye flour.
+//This file contains pastries that don't fit any existing categories.
+////////////////////////////////////////////DONUTS////////////////////////////////////////////
+
+/obj/item/food/donut
+	name = "donut"
+	desc = "Goes great with robust coffee."
+	icon = 'icons/obj/food/donuts.dmi'
+	icon_state = "donut"
+	bite_consumption = 5
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/sugar = 3)
+	tastes = list("donut" = 1)
+	foodtypes = JUNKFOOD | GRAIN | FRIED | SUGAR | BREAKFAST
+	var/decorated_icon = "donut_homer"
+	var/is_decorated = FALSE
+	var/extra_reagent = null
+	var/decorated_adjective = "sprinkled"
+
+/obj/item/food/donut/Initialize()
+	. = ..()
+	AddElement(/datum/element/dunkable, 10)
+	if(prob(30))
```

Comment:
While you're refactoring this code--move this 30 out to a define.

Reply:
I'll move you out to a define

Reply:
#define BEST_MOTH "Mothblocks"